src/physics/base/Model.hh

```diff
@@ -52,7 +52,7 @@ class Model
 
   public:
     // Virtual destructor for polymorphic deletion
-    virtual ~Model() = 0;
+    virtual ~Model();
```

Comment:
Out of curiosity what was the warning message related to this change?

Reply:
This  one is from `-W-weak-vtables`:
```
../src/physics/base/Process.hh:41:7: warning: 'Process' has no out-of-line virtual method definitions; its vtable will be emitted in every translation unit [-Wweak-vtables]
class Process
      ^
```

See https://stackoverflow.com/questions/28786473/clang-no-out-of-line-virtual-method-definitions-pure-abstract-c-class  .

My change might not be  the best solution --  in fact it might prohibit some optimizations because the "non-pure" destructor could have side effects that the compiler can't account for. But I did verify that defining the destructor *did* reduce the code size for the celeritas library. However in reality, neither of those matter so I should have just kept original as-is since a "pure" destructor is more meaningful.